python/proton/_handlers.py

```diff
@@ -950,11 +962,19 @@ class TransactionalClientHandler(MessagingHandler, TransactionHandler):
     and provides a convenience method :meth:`accept` for performing
     a transactional acceptance of received messages.
 
+    .. note:: Sender **auto-settlement** only occurs for a delivery after the
+        sender receives a settled disposition for that delivery. Otherwise,
+        there would be no way to receive any further events for that delivery
+        (such as the subsequent on_settle message that might be expected when
+        the receiver finally settles the message).
+
```

Comment:
As above delete this para we cover it below under auto settlement.
If we want to explain this point we need to attach the explanation to the explanation of settlement as a concept,.

Reply:
Deleted.